tap.go

```diff
@@ -53,6 +56,7 @@ func (bucket *WalrusBucket) StartDCPFeed(args sgbucket.FeedArguments, callback s
 	}
 
 	go func() {
+		defer close(args.DoneChan)
```

Comment:
We called StartTapFeed just above, which is also going to try to close args.DoneChan when the feed completes.  This looks like a potential source of a panic trying to close an already closed channel.

Reply:
Removed the change to close the DoneChan inside the StartTapFeed above to avoid closing an already closed channel.